book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
```

Comment:
@dasileker  I'll implement them but I think this should be optional, it's not in the [requirements](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_associations.md) since the rentals are going to be made by new Objects. (Rental Class)

Reply:
Hello, as you see they are missing pieces, and since the project is incomplete in some part they are recommended to fix them.